heapster-saw/src/Verifier/SAW/Heapster/SAWTranslation.hs

```diff
@@ -3806,16 +3806,18 @@ debugPrettyPermCtx prxs (ptranss :>: ptrans) =
 translateApply :: String -> OpenTerm -> Mb ctx (DistPerms ps) ->
                   ImpTransM ext blocks tops ret ps ctx OpenTerm
 translateApply nm f perms =
-  do expr_ctx <- itiExprCtx <$> ask
+  do assertPermStackEqM nm perms
+     expr_ctx <- itiExprCtx <$> ask
      arg_membs <- itiPermStackVars <$> ask
      let e_args = RL.map (flip RL.get expr_ctx) arg_membs
      i_args <- itiPermStack <$> ask
      return $
+       {-
        trace ("translateApply for " ++ nm ++ " with perm arguments:\n" ++
               -- renderDoc (list $ debugPrettyPermCtx (mbToProxy perms) i_args)
               -- permPrettyString emptyPPInfo (permTransCtxPerms (mbToProxy perms) i_args)
               permPrettyString emptyPPInfo perms
-             ) $
+             ) $ -}
```

Comment:
Why was this removed? Just curious.

I didn't make comments for them, but I have the same question for all the "X starting..." and "X finished..." traces in this file, the trace in `rsConvertFun`, and the trace in `bvSLt`. Is the answer for all of them just that they you haven't found them useful so far?